### DISCUSSION TITLE
Coloring the column at 80 and 100 characters

### DISCUSSION THREAD

**Discussion by Andras-Csanyi** [LazyVim/LazyVim#4569] [Q&A]
I read through the doc and I haven't found anything saying this explicitly, and I am not experienced enough with NeoVim world to understand clearly implicit things.
I'd like to have the column 80 and 100 with slightly different color. I asked Copilot and it says I should use the below. I am just curious if there is a plugin, already part of LazyVim knows this.

I am not married to either the config below or plugin capability.

```
-- Highlight the 80th and 100th columns
vim.opt.colorcolumn = "80,100"
vim.cmd [[highlight ColorColumn ctermbg=lightgrey guibg=lightgrey]]
```

**Comment by msclp** [comment]
This is the right solution, no worries
